Lib/shutil.py

```diff
@@ -1299,6 +1299,9 @@ def unpack_archive(filename, extract_dir=None, format=None):
     was registered for that extension.
 
     In case none is found, a ValueError is raised.
+
+    If `filter` is given, it is passed to he underlying
```

Comment:
typo

```suggestion
    If `filter` is given, it is passed to the underlying
```

Also, have this paragraph immediately follow the `format` paragraph.

Reply:
There are two paragraphs for `format`, and this follows them.

Reply:
Ah, I see.  I think that second paragraph really should be part of the previous paragraph.  We can make that change later, though.